components/people/PeoplePage.tsx

```diff
@@ -0,0 +1,59 @@
+import ArrowIcon from "../icons/Arrow";
+import PersonCard from "./PersonCard";
+
+import { useContext, useState } from 'react';
+
+const PeoplePage = ({people, setShowMessage}) => {
+    const verifiedPeopleObj = people.filter(person => person.verified === true);
+
+    const reducedPeopleObj = verifiedPeopleObj.slice(0, 6);
+    const [peopleObj, setPeopleObj] = useState(reducedPeopleObj);
+  
+    const unVerifiedPeopleObj = people.filter(person => person.verified !== true);
+
+    const handleShowMore = () => {
+        setPeopleObj([...verifiedPeopleObj, ...unVerifiedPeopleObj]);
```

Comment:
```suggestion
        setPeopleObj(people);
```
The current code is okay, either ways